<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_window</class>
 <widget class="QDialog" name="create_window">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Створення нотатки</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="nameFile">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTextEdit" name="textFile"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="closeBtn">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="createBtn">
     <property name="text">
      <string>Створити</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="notificationRd">
     <property name="text">
      <string>Увімкнути нагадування</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDateEdit" name="dataFile"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
